app/views/layouts/main.hbs

```diff
@@ -24,7 +24,7 @@
     <script src="/js/angular.min.js"></script>
     <script src="/js/core.js"></script>
   </head>
-  <body>    
+  <body class="{{{body_class}}}">
```

Comment:
This is a good way. I'll take a stab at this then pass it back to you for review.